test/services/assets/handlers/file.spec.js

```diff
@@ -30,7 +30,7 @@ describe('services', () => {
         const handler = new FileHandler();
         expect(handler.includeFile('/User/test/Project/.DS_Store')).toBeFalsy();
         expect(handler.includeFile('C:/test/Project/Thumbs.db')).toBeFalsy();
-        expect(handler.includeFile('.statwrap-project.json')).toBeFalsy();
+        expect(handler.includeFile(Constants.StatWrapFiles.PROJECT)).toBeFalsy();
```

Comment:
I want it on the record that I really appreciate you fixing *my* mistake of hard-coding constants.  Thank you :)

Reply:
Not at all, it was just a small thing that caught my attention 😆. We all make mistakes!